### PR TITLE
1421 Have state of date picker for reports in URL

### PR DESCRIPTION
**Pull request by Alastair-L**
### Issue #:
https://github.com/beyondessential/tupaia-backlog/issues/1421

Note: everything but the last commit was reviewed here: https://github.com/beyondessential/tupaia/pull/1614

### CODE REVIEW

File: packages/web-frontend/src/components/View/MatrixWrapper/MatrixWrapper.js [hunk 1]
```diff
@@ -155,15 +155,11 @@ export class MatrixWrapper extends Component {
     this.updateWrapper = this.updateWrapper.bind(this);
   }
 
-  componentDidMount() {
-    const { startDate, endDate } = this.props.viewContent || {};
-    this.props.onSetDateRange(startDate, endDate); // Force a load when the expanded matrix first mounts
-  }
-
```
Comment (Alastair-L) [inline packages/web-frontend/src/components/View/MatrixWrapper/MatrixWrapper.js:162]: This was the only change. The matrix will always load on mount (initiated in `EnlargedDialog`)

**Comment by tcaiger** [comment]
Hey Alistair, I'm not sure I'm following the relationship of this PR to the previous one that I reviewed. Can you comment below with the link to that one? And if it's just that one change, then why so many files changed from dev?

**Comment by Alastair-L** [comment]
Sure @tcaiger. It was this one: #1614

**Review by tcaiger — APPROVED** [review]
Thanks for the clarification Alistair. Looks all good.

**Pull request merged**